source/common/ssl/context_impl.cc

```diff
@@ -356,9 +325,7 @@ std::string ContextImpl::getSerialNumber(X509* cert) {
   if (char_serial_number != nullptr) {
     std::string serial_number(char_serial_number);
     OPENSSL_free(char_serial_number);
-#ifdef OPENSSL_IS_BORINGSSL
     std::transform(serial_number.begin(), serial_number.end(), serial_number.begin(), ::toupper);
```

Comment:
This is only used for admin/debug output. There is no automation. I think we can kill the conversion.